app/javascript/mastodon/components/avatar.js

```diff
@@ -53,10 +54,12 @@ export default class Avatar extends React.PureComponent {
       backgroundSize: `${size}px ${size}px`,
     };
 
-    if (hovering || animate) {
-      style.backgroundImage = `url(${src})`;
-    } else {
-      style.backgroundImage = `url(${staticSrc})`;
+    if (!hidden) {
+      if (hovering || animate) {
+        style.backgroundImage = `url(${src})`;
+      } else {
+        style.backgroundImage = `url(${staticSrc})`;
+      }
```

Comment:
This works in profiles, but looks weird/bad in other contexts:
![image](https://user-images.githubusercontent.com/384364/167248045-748e962e-db34-489a-a13b-9b4de0d79827.png)